interpreters/src/drop/table_dropper/mod.rs

```diff
@@ -0,0 +1,24 @@
+// Copyright 2022 CeresDB Project Authors. Licensed under Apache-2.0.
+
+use std::sync::Arc;
+
+use async_trait::async_trait;
+use sql::plan::DropTablePlan;
+use table_engine::engine::TableEngineRef;
+
+use crate::{context::Context, drop::Result, interpreter::Output};
+
+pub mod catalog_based;
+pub mod meta_based;
+
+pub type TableDropperRef = Arc<dyn TableDropper + Send + Sync>;
+
+#[async_trait]
+pub trait TableDropper {
+    async fn drop_table(
```

Comment:
I think `drop_table` in `TableDropper` may be a bit verbose. Same as `TableCreator`.

Reply:
It's true, and I named them as `drop/create` at first. However, the `drop` method has been used by `Drop` trait, and I have to name it `drop_table` here.

Reply:
Why not make `Manipulator` and place drop and create logic into it?